includes/polling/os/fiberhome-switch.inc.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * LibreNMS - FiberHome Switch device support - OS module
+ *
+ * @category   Network_Monitoring
+ * @package    LibreNMS
+ * @subpackage Fiber Home Switch device support
+ * @author     Christoph Zilian <czilian@hotmail.com>
+ * @license    http://gnu.org/copyleft/gpl.html GNU GPL
+ * @link       https://github.com/librenms/librenms/
+
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ **/
+
+function hexbin($hex_string) // convert the hex OID content to character
```

Comment:
Please move this to `includes/functions.php`

Reply:
done